netlify.toml

```diff
@@ -10,3 +15,7 @@
 
 [context.production.environment]
   GATSBY_ENVIRONMENT_CONTEXT = "production"
+
+[functions]
+  directory = "packages/paste-website/functions"
+  external_node_modules = ["chrome-aws-lambda", "puppeteer-core", "path-match"]
```

Comment:
Whoops, `path-match` can be removed from here and the package.json. It was from the example I was trying to get to work and have since removed